### DISCUSSION TITLE
Can MPC Support QtQuick

### DISCUSSION THREAD

**Discussion by LukeMouse** [DOCGroup/MPC#137] [General]
I managed to manage my qt widget project with MPC, but recently I found QML is more powerful on build UI, however I found no support for QML in MPC-Qt configure,how can I support QML with MPC, any ideas, many thanks!

**Comment by jwillemsen** [comment]
Thanks for the [PR](https://github.com/DOCGroup/MPC/pull/138) to add support for QtQuick
